stable/risk-advisor/templates/deployment.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
```

Comment:
See best practices for standard labels and apply them everywhere:
https://github.com/kubernetes/helm/blob/master/docs/chart_best_practices/labels.md

Use `{{ template "name" . }}` for the `app` label.